main.py

```diff
@@ -483,6 +500,9 @@ async def command_list(message: Message):
     global current_channel_content
     current_channel_content = channel_media_attachments
 
+    global current_bust_content
+    current_bust_content = channel_media_attachments.copy()
```

Comment:
If we do this, i think describing them as "local files" is more fitting than "attachments" in general (and it's how we describe them in other places in the code), since they no longer have any link to Discord structures. So probably `remove_attachments()` should be renamed to `remove_local_files` or something as well

Reply:
Reducing the footprint made the delete operation trivial, so I just folded it into the single call site.